MailChimp.Net/Core/HttpRequestExtensions.cs

```diff
@@ -53,7 +53,9 @@ public static class HttpRequestExtensions
                                                        NullValueHandling.Ignore
                                                }
                                        };
-            return await client.PatchAsync(requestUri, new ObjectContent<T>(value, jsonFormatter)).ConfigureAwait(false);
+            var content = new ObjectContent<T>(value, jsonFormatter);
+            var m = content.ReadAsStringAsync().Result; // This line fixes the tests but I dont know why (ngm)
```

Comment:
Is it possible to do:

```
        private readonly static JsonSerializerSettings jsonSettings = new JsonSerializerSettings() { 
            NullValueHandling = NullValueHandling.Ignore
        };
...
                     new StringContent(JsonConvert.SerializeObject(value, jsonSettings), 
                                                                       Encoding.UTF8, "application/json")
```
In order to fix the problems? Then perhaps you won't need to depend on webapi library.